light-zk.js/src/wallet/createOutUtxos.ts

```diff
@@ -119,15 +119,15 @@ export function createOutUtxos({
       ? publicAmountSol.add(relayerFee)
       : relayerFee;
   }
-
+  // @ts-ignore
```

Comment:
why do we need a @ts-ignore here?

Reply:
<img width="762" alt="Screenshot 2023-07-20 at 10 35 11" src="https://github.com/Lightprotocol/light-protocol/assets/7057784/3f90309c-8a1c-474b-99bb-b7fa3c3725ec">

the variable isn't read

Reply:
let's remove it then

Reply:
removed ✅